GERBIL got stuck because of HTTP observer

After running for several months, GERBIL got stuck in hash map methods of the HTTP observer.

```
eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=100.0% of dataset
org.aksw.gerbil.http.ObservedHttpRequest.equals(ObservedHttpRequest.java:52)
com.carrotsearch.hppc.ObjectLongOpenHashMap.put(ObjectLongOpenHashMap.java:181)
org.aksw.gerbil.http.InterruptingObserver.reportStart(InterruptingObserver.java:94)
org.aksw.gerbil.http.HttpManagement.reportStart(HttpManagement.java:109)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.createGetRequest(AbstractHttpRequestEmitter.java:103)
org.aksw.gerbil.semantic.sameas.impl.http.HTTPBasedSameAsRetriever.requestModel(HTTPBasedSameAsRetriever.java:88)
org.aksw.gerbil.semantic.sameas.impl.http.HTTPBasedSameAsRetriever.retrieveSameURIs(HTTPBasedSameAsRetriever.java:57)
org.aksw.gerbil.semantic.sameas.impl.http.HTTPBasedSameAsRetriever.retrieveSameURIs(HTTPBasedSameAsRetriever.java:178)
org.aksw.gerbil.semantic.sameas.impl.DomainBasedSameAsRetrieverManager.retrieveSameURIs(DomainBasedSameAsRetrieverManager.java:70)
org.aksw.gerbil.semantic.sameas.impl.DomainBasedSameAsRetrieverManager.retrieveSameURIs(DomainBasedSameAsRetrieverManager.java:58)
org.aksw.gerbil.semantic.sameas.impl.UriFilteringSameAsRetrieverDecorator.retrieveSameURIs(UriFilteringSameAsRetrieverDecorator.java:56)
org.aksw.gerbil.semantic.sameas.impl.CrawlingSameAsRetrieverDecorator.addSameURIs(CrawlingSameAsRetrieverDecorator.java:72)
org.aksw.gerbil.semantic.sameas.impl.CrawlingSameAsRetrieverDecorator.retrieveSameURIs(CrawlingSameAsRetrieverDecorator.java:51)
org.aksw.gerbil.semantic.sameas.impl.cache.FileBasedCachingSameAsRetriever.retrieveSameURIs(FileBasedCachingSameAsRetriever.java:130)
org.aksw.gerbil.semantic.sameas.impl.AbstractSameAsRetrieverDecorator.addSameURIs(AbstractSameAsRetrieverDecorator.java:43)
org.aksw.gerbil.semantic.sameas.SameAsRetrieverUtils.addSameURIsToMeanings(SameAsRetrieverUtils.java:39)
org.aksw.gerbil.execute.ExperimentTask.prepareAnnotatorResults(ExperimentTask.java:222)
org.aksw.gerbil.execute.ExperimentTask.runExperiment(ExperimentTask.java:324)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:135)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","A2KB","WEAK_ANNOTATION_MATCH")
state=RUNNABLE
progress=24.05982905982906% of dataset
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.request(NIFBasedAnnotatorWebservice.java:148)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performAnnotation(NIFBasedAnnotatorWebservice.java:109)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performA2KBTask(NIFBasedAnnotatorWebservice.java:89)
org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator.performExtraction(TimeMeasuringAnnotatorDecorator.java:211)
org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator$TimeMeasuringA2KBAnnotator.performA2KBTask(TimeMeasuringAnnotatorDecorator.java:141)
org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator.performExtraction(ErrorCountingAnnotatorDecorator.java:276)
org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator$ErrorCountingA2KBAnnotator.performA2KBTask(ErrorCountingAnnotatorDecorator.java:159)
org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator.performExtraction(SingleInstanceSecuringAnnotatorDecorator.java:243)
org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator$SingleInstanceSecuringA2KBAnnotator.performA2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:147)
org.aksw.gerbil.execute.ExperimentTask.runExperiment(ExperimentTask.java:341)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:135)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","A2KB","WEAK_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","A2KB","WEAK_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:475)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","A2KB","WEAK_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=96.8413496051687% of dataset
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:483)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.request(NIFBasedAnnotatorWebservice.java:148)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performAnnotation(NIFBasedAnnotatorWebservice.java:109)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performD2KBTask(NIFBasedAnnotatorWebservice.java:79)
org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator.performD2KBTask(TimeMeasuringAnnotatorDecorator.java:202)
org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator$TimeMeasuringD2KBAnnotator.performD2KBTask(TimeMeasuringAnnotatorDecorator.java:106)
org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator.performD2KBTask(ErrorCountingAnnotatorDecorator.java:253)
org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator$ErrorCountingD2KBAnnotator.performD2KBTask(ErrorCountingAnnotatorDecorator.java:124)
org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator.performD2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:225)
org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator$SingleInstanceSecuringD2KBAnnotator.performD2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:111)
org.aksw.gerbil.execute.ExperimentTask.runExperiment(ExperimentTask.java:317)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:135)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:475)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","A2KB","WEAK_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:475)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=96.05683836589698% of dataset
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.request(NIFBasedAnnotatorWebservice.java:148)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performAnnotation(NIFBasedAnnotatorWebservice.java:109)
org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performD2KBTask(NIFBasedAnnotatorWebservice.java:79)
org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator.performD2KBTask(TimeMeasuringAnnotatorDecorator.java:202)
org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator$TimeMeasuringD2KBAnnotator.performD2KBTask(TimeMeasuringAnnotatorDecorator.java:106)
org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator.performD2KBTask(ErrorCountingAnnotatorDecorator.java:253)
org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator$ErrorCountingD2KBAnnotator.performD2KBTask(ErrorCountingAnnotatorDecorator.java:124)
org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator.performD2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:225)
org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator$SingleInstanceSecuringD2KBAnnotator.performD2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:111)
org.aksw.gerbil.execute.ExperimentTask.runExperiment(ExperimentTask.java:317)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:135)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","A2KB","WEAK_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:475)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:475)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:475)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)

eTConfig("XXX","XXX","D2KB","STRONG_ANNOTATION_MATCH")
state=RUNNABLE
progress=null
com.carrotsearch.hppc.Internals.rehash(Internals.java:10)
com.carrotsearch.hppc.ObjectLongOpenHashMap.shiftConflictingKeys(ObjectLongOpenHashMap.java:462)
com.carrotsearch.hppc.ObjectLongOpenHashMap.remove(ObjectLongOpenHashMap.java:438)
org.aksw.gerbil.http.InterruptingObserver.reportEnd(InterruptingObserver.java:106)
org.aksw.gerbil.http.HttpManagement.reportEnd(HttpManagement.java:149)
org.aksw.gerbil.http.AbstractHttpRequestEmitter.closeRequest(AbstractHttpRequestEmitter.java:124)
org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker.entityExists(HttpBasedEntityChecker.java:75)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkUri(EntityCheckerManagerImpl.java:135)
org.aksw.gerbil.dataset.check.impl.FileBasedCachingEntityCheckerManager.checkUri(FileBasedCachingEntityCheckerManager.java:154)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMeaning(EntityCheckerManagerImpl.java:84)
org.aksw.gerbil.dataset.check.impl.EntityCheckerManagerImpl.checkMarkings(EntityCheckerManagerImpl.java:64)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getPreparedDataset(AbstractDatasetConfiguration.java:79)
org.aksw.gerbil.dataset.AbstractDatasetConfiguration.getDataset(AbstractDatasetConfiguration.java:50)
org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:101)
org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
java.lang.Thread.run(Thread.java:745)
```

This might be caused by many requests that were observed but not removed from the hash map.

From the tail of the log, it can be seen that the observer encountered an error. The problem might be related to dexter not answering a request.
```
2017-01-25 01:03:54,680 [pool-21-thread-30] ERROR [org.aksw.gerbil.http.InterruptingObserver] - <Tried to remove an observed request that is not existing (ObservedHttpRequest [request=HEAD http://dbpedia.org/resource/OpenDarwin.org HTTP/1.1, emitter=org.aksw.gerbil.dataset.check.impl.HttpBasedEntityChecker@3d354834]). This is a fatal error.>
2017-01-25 01:04:51,180 [pool-21-thread-9] ERROR [org.aksw.gerbil.http.AbstractHttpRequestEmitter] - <Exception while sending request.>
org.apache.http.conn.ConnectTimeoutException: Connect to dexterdemo.isti.cnr.it:8080 [dexterdemo.isti.cnr.it/146.48.82.49] failed: Connection timed out
        at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:156)
        at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
        at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
        at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
        at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
        at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
        at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
        at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:107)
        at org.aksw.gerbil.http.AbstractHttpRequestEmitter.sendRequest(AbstractHttpRequestEmitter.java:151)
        at org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.request(NIFBasedAnnotatorWebservice.java:137)
        at org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performAnnotation(NIFBasedAnnotatorWebservice.java:109)
        at org.aksw.gerbil.annotator.impl.nif.NIFBasedAnnotatorWebservice.performA2KBTask(NIFBasedAnnotatorWebservice.java:89)
        at org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator.performExtraction(TimeMeasuringAnnotatorDecorator.java:211)
        at org.aksw.gerbil.annotator.decorator.TimeMeasuringAnnotatorDecorator$TimeMeasuringA2KBAnnotator.performA2KBTask(TimeMeasuringAnnotatorDecorator.java:141)
        at org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator.performExtraction(ErrorCountingAnnotatorDecorator.java:276)
        at org.aksw.gerbil.annotator.decorator.ErrorCountingAnnotatorDecorator$ErrorCountingA2KBAnnotator.performA2KBTask(ErrorCountingAnnotatorDecorator.java:159)
        at org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator.performExtraction(SingleInstanceSecuringAnnotatorDecorator.java:243)
        at org.aksw.gerbil.annotator.decorator.SingleInstanceSecuringAnnotatorDecorator$SingleInstanceSecuringA2KBAnnotator.performA2KBTask(SingleInstanceSecuringAnnotatorDecorator.java:147)
        at org.aksw.gerbil.execute.ExperimentTask.runExperiment(ExperimentTask.java:341)
        at org.aksw.gerbil.execute.ExperimentTask.run(ExperimentTask.java:135)
        at org.aksw.simba.topicmodeling.concurrent.workers.WorkerImpl.run(WorkerImpl.java:44)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.ConnectException: Connection timed out
        at java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:589)
        at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
        at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:141)
        ... 25 more
```

--- a/src/main/java/org/aksw/gerbil/http/HttpManagement.java
+++ b/src/main/java/org/aksw/gerbil/http/HttpManagement.java
@@ -61,7 +61,28 @@ public class HttpManagement {
     private static final String USER_AGENT_STRING = "GERBIL/" + GerbilConfiguration.getGerbilVersion()
             + " (http://aksw.org/Projects/GERBIL.html)";
 
+    /**
+     * Singleton instance.
+     */
     private static HttpManagement instance;
+
+    /**
+     * The interrupting observer used to interrupt requests that take too long.
+     */
+    protected InterruptingObserver interruptingObserver;
+    /**
+     * The HTTP client used for all HTTP communication.
+     */
+    protected CloseableHttpClient client;
+    /**
+     * The usage agent string of this program.
+     */
+    protected String userAgent;
+    /**
+     * The map of domains that could block this client and the last time at which
+     * they have been accessed.
+     */
+    protected Map<String, Long> blockingDomainTimestampMapping = new HashMap<>();
 
     public synchronized static HttpManagement getInstance() {
         if (instance == null) {
@@ -93,14 +114,9 @@ public class HttpManagement {
         return instance;
     }
 
-    protected InterruptingObserver interruptingObserver;
-    protected CloseableHttpClient client;
-    protected String userAgent;
-    protected Map<String, Long> blockingDomainTimestampMapping = new HashMap<>();
-
     protected HttpManagement(InterruptingObserver interruptingObserver, String userAgent) {
         this.interruptingObserver = interruptingObserver;
-        this.client = generateHttpClientBuilder().build();
+        this.client = generateHttpClientBuilder().setUserAgent(userAgent).build();
     }
 
     public void reportStart(HttpRequestEmitter emitter, HttpUriRequest request) {

--- a/src/main/java/org/aksw/gerbil/http/HttpManagement.java
+++ b/src/main/java/org/aksw/gerbil/http/HttpManagement.java
@@ -116,7 +116,8 @@ public class HttpManagement {
 
     protected HttpManagement(InterruptingObserver interruptingObserver, String userAgent) {
         this.interruptingObserver = interruptingObserver;
-        this.client = generateHttpClientBuilder().setUserAgent(userAgent).build();
+        this.userAgent = userAgent;
+        this.client = generateHttpClientBuilder().build();
     }
 
     public void reportStart(HttpRequestEmitter emitter, HttpUriRequest request) {

--- a/src/main/java/org/aksw/gerbil/http/HttpManagement.java
+++ b/src/main/java/org/aksw/gerbil/http/HttpManagement.java
@@ -16,7 +16,8 @@
  */
 package org.aksw.gerbil.http;
 
-import java.util.concurrent.Semaphore;
+import java.util.HashMap;
+import java.util.Map;
 
 import org.aksw.gerbil.config.GerbilConfiguration;
 import org.apache.http.HttpHost;
@@ -26,8 +27,6 @@ import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.impl.conn.DefaultProxyRoutePlanner;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
-import com.carrotsearch.hppc.ObjectLongOpenHashMap;
 
 public class HttpManagement {
 
@@ -97,8 +96,7 @@ public class HttpManagement {
     protected InterruptingObserver interruptingObserver;
     protected CloseableHttpClient client;
     protected String userAgent;
-    protected Semaphore blockingDomainMappingMutex = new Semaphore(1);
-    protected ObjectLongOpenHashMap<String> blockingDomainTimestampMapping = new ObjectLongOpenHashMap<String>();
+    protected Map<String, Long> blockingDomainTimestampMapping = new HashMap<>();
 
     protected HttpManagement(InterruptingObserver interruptingObserver, String userAgent) {
         this.interruptingObserver = interruptingObserver;
@@ -112,30 +110,20 @@ public class HttpManagement {
     }
 
     protected void getStartPermission(HttpUriRequest request) {
-        try {
-            blockingDomainMappingMutex.acquire();
-        } catch (InterruptedException e) {
-            LOGGER.error("Interrupted while waiting for mutex to access the list of blocking domains. Aborting.");
-            return;
-        }
         long timeToSleep = 0;
-        try {
+        synchronized (blockingDomainTimestampMapping) {
             String host = request.getURI().getHost();
             if ((host == null) || (!blockingDomainTimestampMapping.containsKey(host))) {
                 return;
             }
-            // we are allowed to use lget and lset since the mutex is securing
-            // the hashmap
-            long lastRequestTimeStamp = blockingDomainTimestampMapping.lget();
+            long lastRequestTimeStamp = blockingDomainTimestampMapping.get(host);
             long currentTime = System.currentTimeMillis();
             timeToSleep = BLOCKING_DOMAIN_WAITING_TIME - (currentTime - lastRequestTimeStamp);
             if (timeToSleep > 0) {
-                blockingDomainTimestampMapping.lset(currentTime + timeToSleep);
+                blockingDomainTimestampMapping.put(host, currentTime + timeToSleep);
             } else {
-                blockingDomainTimestampMapping.lset(currentTime);
+                blockingDomainTimestampMapping.put(host, currentTime);
             }
-        } finally {
-            blockingDomainMappingMutex.release();
         }
 
         if (timeToSleep > 0) {
@@ -176,16 +164,8 @@ public class HttpManagement {
      * requests.
      */
     public void addBlockingDomain(String domain) {
-        try {
-            blockingDomainMappingMutex.acquire();
-        } catch (InterruptedException e) {
-            LOGGER.error("Interrupted while waiting for mutex to access the list of blocking domains. Aborting.");
-            return;
-        }
-        try {
-            blockingDomainTimestampMapping.put(domain, 0);
-        } finally {
-            blockingDomainMappingMutex.release();
+        synchronized (blockingDomainTimestampMapping) {
+            blockingDomainTimestampMapping.put(domain, 0L);
         }
     }
 

--- a/src/main/java/org/aksw/gerbil/http/InterruptingObserver.java
+++ b/src/main/java/org/aksw/gerbil/http/InterruptingObserver.java
@@ -17,6 +17,7 @@
 package org.aksw.gerbil.http;
 
 import java.util.HashMap;
+import java.util.Iterator;
 import java.util.Map;
 import java.util.Map.Entry;
 
@@ -54,14 +55,16 @@ public class InterruptingObserver implements Runnable {
             long waitingTime;
             long currentTime = System.currentTimeMillis();
             ObservedHttpRequest observedRequest;
-            for (Entry<ObservedHttpRequest, Long> e : observedRequests.entrySet()) {
-                waitingTime = currentTime - e.getValue();
+            Iterator<Entry<ObservedHttpRequest, Long>> iterator = observedRequests.entrySet().iterator();
+            while (iterator.hasNext()) {
+                Entry<ObservedHttpRequest, Long> entry = iterator.next();
+                waitingTime = currentTime - entry.getValue();
                 if (waitingTime > maxWaitingTime) {
-                    observedRequest = e.getKey();
+                    observedRequest = entry.getKey();
                     if (observedRequest.request.isAborted()) {
                         LOGGER.warn(
                                 "Observing an HTTP request that is already aborted. This could be mean that the HTTP management does not have been informed about the termination of a request.");
-                        observedRequests.remove(observedRequest);
+                        iterator.remove();
                     } else {
                         LOGGER.info("The HTTP request emitter \"{}\" already runs for {} ms. Trying to interrupt it.",
                                 observedRequest.emitter.getName(), waitingTime);

--- a/src/main/java/org/aksw/gerbil/http/InterruptingObserver.java
+++ b/src/main/java/org/aksw/gerbil/http/InterruptingObserver.java
@@ -69,8 +69,6 @@ public class InterruptingObserver implements Runnable {
                     }
                 }
             }
-        } finally {
-            observedMappingMutex.release();
         }
     }
 

--- a/src/main/java/org/aksw/gerbil/http/InterruptingObserver.java
+++ b/src/main/java/org/aksw/gerbil/http/InterruptingObserver.java
@@ -16,13 +16,13 @@
  */
 package org.aksw.gerbil.http;
 
-import java.util.concurrent.Semaphore;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
 
 import org.apache.http.client.methods.HttpUriRequest;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
-import com.carrotsearch.hppc.ObjectLongOpenHashMap;
 
 public class InterruptingObserver implements Runnable {
 
@@ -30,8 +30,7 @@ public class InterruptingObserver implements Runnable {
 
     private long maxWaitingTime;
     private long checkInterval;
-    private final ObjectLongOpenHashMap<ObservedHttpRequest> observedRequests = new ObjectLongOpenHashMap<ObservedHttpRequest>();
-    private final Semaphore observedMappingMutex = new Semaphore(1);
+    private final Map<ObservedHttpRequest, Long> observedRequests = new HashMap<>();
 
     public InterruptingObserver(long maxWaitingTime, long checkInterval) {
         this.maxWaitingTime = maxWaitingTime;
@@ -51,25 +50,19 @@ public class InterruptingObserver implements Runnable {
     }
 
     private void checkAnnotators() {
-        try {
-            observedMappingMutex.acquire();
-        } catch (InterruptedException e) {
-            LOGGER.error("Interrupted while waiting for mutex. Aborting.");
-        }
-
-        long waitingTime;
-        long currentTime = System.currentTimeMillis();
-        ObservedHttpRequest observedRequest;
-        for (int i = 0; i < observedRequests.allocated.length; ++i) {
-            if (observedRequests.allocated[i]) {
-                waitingTime = currentTime - observedRequests.values[i];
+        synchronized (observedRequests) {
+            long waitingTime;
+            long currentTime = System.currentTimeMillis();
+            ObservedHttpRequest observedRequest;
+            for (Entry<ObservedHttpRequest, Long> e : observedRequests.entrySet()) {
+                waitingTime = currentTime - e.getValue();
                 if (waitingTime > maxWaitingTime) {
-                    observedRequest = ((ObservedHttpRequest) ((Object[]) observedRequests.keys)[i]);
+                    observedRequest = e.getKey();
                     LOGGER.info("The HTTP request emitter \"{}\" already runs for {} ms. Trying to interrupt it.",
                             observedRequest.emitter.getName(), waitingTime);
                     try {
                         observedRequest.emitter.interrupt(observedRequest.request);
-                    } catch (UnsupportedOperationException e) {
+                    } catch (UnsupportedOperationException ex) {
                         LOGGER.error("Couldn't interrupt request of HTTP request emitter \""
                                 + observedRequest.emitter.getName() + "\" that is already running for " + waitingTime
                                 + " ms.");
@@ -77,38 +70,29 @@ public class InterruptingObserver implements Runnable {
                 }
             }
         }
-        observedMappingMutex.release();
     }
 
     public void reportStart(HttpRequestEmitter emitter, HttpUriRequest request) {
         ObservedHttpRequest observedRequest = new ObservedHttpRequest(request, emitter);
-        try {
-            observedMappingMutex.acquire();
-        } catch (InterruptedException e) {
-            LOGGER.error("Interrupted while waiting for mutex. Aborting.");
+        synchronized (observedRequests) {
+            if (observedRequests.containsKey(observedRequest)) {
+                LOGGER.error("There already is an observed request equal to this new one (" + observedRequest.toString()
+                        + "). Note that this is a fatal error and the old request will be overwritten.");
+            }
+            observedRequests.put(observedRequest, System.currentTimeMillis());
         }
-        if (observedRequests.containsKey(observedRequest)) {
-            LOGGER.error("There already is an observed request equal to this new one (" + observedRequest.toString()
-                    + "). Note that this is a fatal error and the old request will be overwritten.");
-        }
-        observedRequests.put(observedRequest, System.currentTimeMillis());
-        observedMappingMutex.release();
     }
 
     public void reportEnd(HttpRequestEmitter emitter, HttpUriRequest request) {
         ObservedHttpRequest observedRequest = new ObservedHttpRequest(request, emitter);
-        try {
-            observedMappingMutex.acquire();
-        } catch (InterruptedException e) {
-            LOGGER.error("Interrupted while waiting for mutex. Aborting.");
+        synchronized (observedRequests) {
+            if (observedRequests.containsKey(observedRequest)) {
+                observedRequests.remove(observedRequest);
+            } else {
+                LOGGER.error("Tried to remove an observed request that is not existing (" + observedRequest.toString()
+                        + "). This is a fatal error.");
+            }
         }
-        if (observedRequests.containsKey(observedRequest)) {
-            observedRequests.remove(observedRequest);
-        } else {
-            LOGGER.error("Tried to remove an observed request that is not existing (" + observedRequest.toString()
-                    + "). This is a fatal error.");
-        }
-        observedMappingMutex.release();
     }
 
     public long getMaxWaitingTime() {

